joint_cmd[6] joint_commands
int32 velocity
